### PR TITLE
Switch to sass

### PR DESCRIPTION
**Pull request by kennedy**
This is dependent on #8 , please merge that one first :)

- This PR will remove unnecessary includes because the default layout is already refactored
- Adds a License thats CC0
- Remove placeholder images

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
-    <li><a href="/categories" class="button big">Tags</a></li>
-    <li><a href="/recipes" class="button big">Browse All</a></li>
-    <li><a href="https://github.com/thenaterhood/fossrecipes" class="button big">Github</a></li>
-</ul>
+<!-- Banner -->
+<section id="banner">
+    <div class="content">
+        <header>
+            <h1>{{ page.title }} </h1>
+            <p>A free and fully responsive site template</p>
```
Comment (thenaterhood) [inline index.html]: Looks like something from the default template that should be removed or changed.

File: index.html [hunk 2]
```diff
@@ -9,12 +9,12 @@
     <div class="content">
         <header>
             <h1>{{ page.title }} </h1>
-            <p>A free and fully responsive site template</p>
+            <p>Free and community-sourced recipes.</p>
```
Comment (kennedy) [inline index.html:12]: Fixed!

**Comment by kennedy** [comment]
hacktoberfest-accepted

File: index.html (hunk 2 above)
Comment (thenaterhood) [reply index.html:12]: Isn't that redundant with the paragraph directly below the header now?

**Review by thenaterhood — APPROVED** [review]

**Pull request merged**